docs/user/custom-domains.rst

```diff
@@ -2,125 +2,41 @@ Custom Domains
 
 * ``<slug>.readthedocs.io`` for |org_brand|
 * ``<slug>.readthedocs-hosted.com`` for |com_brand|.
 
-For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
+How does it work?
+-----------------
 
-.. contents:: Contents
-    :local:
+To use a custom domain, you enter the domain in your Read the Docs project's administration and update your DNS provider with a new DNS entry.
 
-Adding a custom domain
-----------------------
+These two actions are all that are needed. Once the DNS record has propagated, Read the Docs automatically issues an SSL certificate through Cloudflare and starts serving your documentation.
 
-To setup your custom domain, follow these steps:
+In case you change your domain name, your documentation can have multiple secondary domains but only one **canonical** domain name.
+Additional domains or subdomains will redirect to the canonical domain.
 
-#. Go the :guilabel:`Admin` tab of your project.
-#. Click on :guilabel:`Domains`.
-#. Enter the domain where you want to serve the documentation from (e.g. ``docs.example.com``).
-#. Mark the :guilabel:`Canonical` option if you want use this domain
-   as your :doc:`canonical domain </canonical-urls>`.
-#. Click on :guilabel:`Add`.
-#. At the top of the next page you'll find the value of the DNS record that you need to point your domain to.
-   For |org_brand| this is ``readthedocs.io``, and for :doc:`/commercial/index`
-   the record is in the form of ``<hash>.domains.readthedocs.com``.
+To make this work, Read the Docs generates a special text that you are responsible for copy-pasting to your domain's DNS.
+In most cases, the ``CNAME`` record is used.
+This is all that's needed for a web browser to resolve your domain name to Read the Docs' servers and for our servers to match the right documentation project.
+You will find all the details in :doc:`/guides/custom-domains`.
 
-   .. note::
 
-      For a subdomain like ``docs.example.com`` add a CNAME record,
-      and for a root domain like ``example.com`` use an ANAME or ALIAS record.
+What to consider
+----------------
 
-By default, we provide a validated SSL certificate for the domain,
-managed by `Cloudflare <https://www.cloudflare.com/>`_.
-The SSL certificate issuance should happen within a few minutes,
-but might take up to one hour.
-See `SSL certificate issue delays`_ for more troubleshooting options.
+Some Open Source projects have seen their domains expire. Even prominent ones.
+It's important that you allocate the responsibility of your domain to a reliable actor in your organization.
+The domain will feature in search indexing and if you lose the domain, domain sharks may exploit this.
 
-As an example, our blog's DNS record looks like this:
+You can choose if the domain is just an alias.
```

Comment:
We should explain how to do this, or remove it... I think we're saying the same thing as above?

Reply:
I swapped it around. "canonical domain" was also used before it got explained and point about search indexing was repeated :+1: